### PR TITLE
Merge routes in Router::Route

### LINKED ISSUE #11: Router::route doesn't merge `MethodRouterOperations`
In the following scenario:

```rust
axum_integration::Router::new()
            .route(
                "/subscriptions",
                post(openapi_handler!(subscriptions::create_subscription)),
            )
            .route(
                "/subscriptions",
                get(openapi_handler!(subscriptions::list_subscriptions)),
            )
```

The latter `route` invocation will overwrite the former. This behavior is different from axum, as described here https://docs.rs/axum/latest/axum/struct.Router.html#accepting-multiple-methods

### PR DESCRIPTION
**Pull request by slinkydeveloper**
Fix #11

### CODE REVIEW

**Comment by slinkydeveloper** [comment]
@Flowneee

**Comment by slinkydeveloper** [comment]
Added the changelog, also for #13 :)

File: okapi-operation/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 All notable changes to this project will be documented in the changelog of the respective crates.
 This project follows the [Semantic Versioning standard](https://semver.org/).
 
+## [0.2.1] - 2023-05-09
```
Comment (Flowneee) [inline okapi-operation/CHANGELOG.md:5]: mark this as `[Unreleased]` for now pls, rest is LGTM

**Review by Flowneee — CHANGES_REQUESTED** [review]

**Comment by slinkydeveloper** [comment]
Hey, any chance we can merge and release this?

**Pull request merged**